### PR TITLE
[hlm-707]: Common library for shared data classes and other fixes. 

### PR DESCRIPTION
**Pull request by shivab-egov**
- updated build-config.yml
- changed few names. 
- added libraries folder with common-services for shared data classes.

### CODE REVIEW

File: digit-health-services/libraries/services-common/libraries.iml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jayantp-egov) [inline digit-health-services/libraries/services-common/libraries.iml:1]: Remove this file. It should not be there in your commit. Any commit.

File: build/build-config.yml [hunk 2]
```diff
@@ -25,3 +25,18 @@ config:
...
+    build:
+      - work-dir: "digit-health-services/health-delivery-service"
+        image-name: "health-delivery-service"
+        dockerfile: "build/maven/Dockerfile"
+  - name: "builds/health-campaign-services/digit-health-services/health-registration-service"
+    build:
+      - work-dir: "digit-health-services/health-registration-service"
+        image-name: "health-registration-service"
+        dockerfile: "build/maven/Dockerfile"
+  - name: "builds/health-campaign-services/digit-health-services/libraries/services-common"
```
Comment (jayantp-egov) [inline build/build-config.yml:38]: make it health-services-common

File: build/build-config.yml [hunk 3]
```diff
@@ -25,3 +25,18 @@ config:
...
+    build:
+      - work-dir: "digit-health-services/libraries/services-common"
+        image-name: "health-services-common"
+        dockerfile: "build/maven/Dockerfile"
```
Comment (jayantp-egov) [inline build/build-config.yml:42]: Docker file not required

File: digit-health-services/libraries/services-common/pom.xml [hunk 4]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.digit.common</groupId>
+    <artifactId>services-common</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <name>services-common</name>
```
Comment (jayantp-egov) [inline digit-health-services/libraries/services-common/pom.xml:10]: health-services-common

**Review by jayantp-egov — CHANGES_REQUESTED** [review]

File: digit-health-services/libraries/services-common/libraries.iml (hunk 1 above)
Comment (shivab-egov) [reply digit-health-services/libraries/services-common/libraries.iml]: removed.

File: build/build-config.yml (hunk 2 above)
Comment (shivab-egov) [reply build/build-config.yml]: updated to health-services-common

File: digit-health-services/libraries/services-common/pom.xml (hunk 4 above)
Comment (shivab-egov) [reply digit-health-services/libraries/services-common/pom.xml]: updated artifactId, name to health-services-common. Updated version to 1.0.0 and added packaging tag.

**Review by jayantp-egov — APPROVED** [review]

**Pull request merged**